src/controllers/imposterController.js

```diff
@@ -45,6 +45,100 @@ function create (imposters) {
         });
     }
 
+    function recordImposter (saveImposter) {
+        var mountebank = require('../mountebank');
+        var saveFile = mountebank.saveImpostersFile;
+        var saveFileFlag = mountebank.saveImpostersFileFlag;
+
+        if ((saveFileFlag) && (saveFile !== undefined)) {
+            var makeString = saveFile.toString();
```

Comment:
saveFile already is a string, no need to convert

Reply:
Sure, we'll fix that.